packages/enterprise/src/license.ts

```diff
@@ -618,6 +684,8 @@ const lock = new AsyncLock();
 const keyToLicenseData: Record<string, Partial<LicenseInterface>> = {};
 const keyToCacheDate: Record<string, Date> = {};
 
+export let backgroundUpdateLicenseFromServerForTests: Promise<void | LicenseInterface>;
```

Comment:
Does this need to be exported? I don't see it being used elsewhere

Reply:
Yes, as the name applied it is just used in license.test.ts.